Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -862,14 +862,13 @@ class BrowserViewController: UIViewController {
 
     override func observeValueForKeyPath(keyPath: String?, ofObject object: AnyObject?, change: [String: AnyObject]?, context: UnsafeMutablePointer<Void>) {
         let webView = object as! WKWebView
-        if webView !== tabManager.selectedTab?.webView {
-            return
-        }
         guard let path = keyPath else { assertionFailure("Unhandled KVO key: \(keyPath)"); return }
         switch path {
         case KVOEstimatedProgress:
-            guard let progress = change?[NSKeyValueChangeNewKey] as? Float else { break }
-            urlBar.updateProgressBar(progress)
+            if webView == tabManager.selectedTab?.webView {
+                guard let progress = change?[NSKeyValueChangeNewKey] as? Float else { break }
```

Comment:
> looks like commas are allowed only if there are multiple let statements

Try putting reversing the conditions so that the `let` comes second. Not sure where this is actually documented, but I've discovered this through past experimentation!